src/frontend/komponenter/Felleskomponenter/HiddenIf/hiddenIf.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export interface HiddenProps {
+    hidden?: boolean;
+}
+
+export default function hiddenIf<PROPS>(
+    Component: React.ComponentType<PROPS>
+): React.ComponentType<PROPS & HiddenProps> {
+    return (props: PROPS & HiddenProps) => {
+        const { hidden, ...rest } = props as any;
+        if (hidden) {
+            return null;
+        }
+        return <Component {...rest} />;
+    };
+}
```

Comment:
Bra du fikser dette, for jeg skjønner det ikke helt, men kult at det funker! :D

Reply:
https://reactjs.org/docs/higher-order-components.html ;)